powersimdata/design/transmission/mwmiles.py

```diff
@@ -75,6 +86,6 @@ def _calculate_mw_miles(original_grid, ct, exclude_branches=None):
             upgrades["transformer_mw"] += upgraded_capacity
             upgrades["num_transformers"] += 1
         else:
-            raise Exception("Unknown branch: " + str(b))
+            raise Exception("Unknown branch type: " + str(b))
```

Comment:
Would it be helpful to include the unknown branch type `device_type` in the error message and clarify that `b` is the branch id?

Reply:
This else is more for debugging than anything else, since the code should never in normal operation get here. There are no branch device types in our current grid model that are not covered by these three options.